setup.py

```diff
@@ -85,7 +86,7 @@ def run(self):
     version=__version__,
     author="Jose Agraz, Vinayak Ahluwalia, Bhakti Baheti, Spyridon Bakas, Ujjwal Baid, Megh Bhalerao, Brandon Edwards, Karol Gotkowski, Caleb Grenko, Orhun Güley, Ibrahim Ethem Hamamci, Sarthak Pati, Micah Sheller, Juliia Skobleva, Siddhesh Thakur, Spiros Thermos",  # alphabetical order
     author_email="software@cbica.upenn.edu",
-    python_requires=">=3.6",
+    python_requires=">=3.7",
```

Comment:
Do we have package clashes here?

Reply:
Yes, for numpy.

Reply:
Cool. We might need a workaround becaue I would like to support from 3.6--3.10 but not something to worry about now.

Reply:
Alright! Yes, it is unfortunate but there is a major serialization bug with numpy < 1.21, and numpy >=1.21 is for python >=3.7.